<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="12,1">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Filter...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFilmButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::Batched</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="filmInfoWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Name</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Date</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Rate</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Votes</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Genre</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Duration</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Type</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Certificate</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Nudity</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Violence</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenu::item:disabled {
	color: grey;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCertificates"/>
    <addaction name="actionFilters"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCertificates">
   <property name="text">
    <string>Certificates</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>clickPushButton()</signal>
 </slots>
</ui>
